warehouse/manage/views.py

```diff
@@ -1077,6 +1150,23 @@ def create_organization(self):
         return self.default_response
 
 
+@view_config(
+    route_name="manage.organization.roles",
+    context=Organization,
+    renderer="manage/organization/roles.html",
+    uses_session=True,
+    require_methods=False,
+    # permission="manage:organization",
```

Comment:
I forget what status of permissions are currently.... I assume we need to address them more holistically at a later time.

Reply:
Ah, I see these are handled in 11342